File: Push method in Stack/BY using C/Push.c
#include <stdio.h>
#define MAX 5

int stack[MAX]; // Array to hold stack elements
int top = -1;   // Initialize top to -1 indicating the stack is empty

void push(int value)
{
    if (top == MAX - 1) // Check for stack overflow
    {
        printf("Stack Overflow\n");
    }
    else
    {
        top++; // Increment top to point to the next empty position
        stack[top] = value; // Add the new value to the stack
        printf("%d pushed into stack\n", value); // Confirmation message
    }
}

int main()
{
    push(10); // Push 10 onto the stack
    push(20); // Push 20 onto the stack
    push(30); // Push 30 onto the stack

    return 0; // End of the program
}

File: Push method in Stack/Push.java
import java.util.Scanner;

public class Push {

    static final int MAX = 100;
    int top;
    int[] a = new int[MAX];

    Push() {
        top = -1;
    }

    boolean isEmpty() {
        return (top < 0);
    }

    boolean push(int x) {
        if (top >= (MAX - 1)) {
            System.out.println("Stack Overflow");
            return false;
        } else {
            a[++top] = x;
            System.out.println(x + " pushed into stack");
            return true;
        }
    }

    public static void main(String[] args) {
        Push s = new Push();
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter number of elements to push: ");
        int n = sc.nextInt();

        System.out.println("Enter " + n + " elements:");
        for (int i = 0; i < n; i++) {
            s.push(sc.nextInt());
        }
    }
}

    
